<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shapes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>270</y>
      <width>184</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="clear_button_circle">
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="compute_button_circle">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Compute</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>461</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="radius_input"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="radius_label">
         <property name="text">
          <string>Radius:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="result_label">
         <property name="text">
          <string>Result:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="result_text">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>461</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLineEdit" name="height_rectangle_input"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="width_input"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="employees_name_label_2">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="rate_label_2">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="result_label_2">
         <property name="text">
          <string>Result:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="result_text_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>270</y>
      <width>184</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="clear_button_rectangle">
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="compute_button_rectangle">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Compute</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectangle</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>66</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>590</y>
      <width>184</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="clear_button_triangle">
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="compute_button_triangle">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Compute</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>461</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLineEdit" name="side_input"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="employees_name_label_3">
         <property name="text">
          <string>Side:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="result_label_3">
         <property name="text">
          <string>Result:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="result_text_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>461</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="employees_name_label_4">
         <property name="text">
          <string>Base:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="height_triangle_input"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="base_input"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="rate_label_4">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="result_label_4">
         <property name="text">
          <string>Result:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="result_text_4">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>590</y>
      <width>184</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="clear_button_square">
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="compute_button_square">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Compute</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>330</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Right Triangle</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
